iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/voter/CommonAffSectionWordsVoter.java

```diff
@@ -90,6 +101,9 @@ public CommonAffSectionWordsVoter(List<Character> charsToFilter, int wordToRemov
     @Override
     public boolean voteMatch(AffMatchAffiliation affiliation, AffMatchOrganization organization) {
         
+        if (isCountryCodeDefinedAndDifferent(affiliation, organization)) {
+            return false;
+        }
```

Comment:
One thing that I only remembered while doing dishes: this criterion should definitely be added to the class documentation.

Reply:
> Not remembering much about this codebase, at first I thought that putting this check in this one particular voter looks weird and certainly it should be its own voter - but now I see that there are apparently no veto votes so this would sadly make no sense.

100% agree. When I was introducing `isCountryCodeDefinedAndDifferent()` method it crossed my mind we should introduce veto votes as a generic solution applicable to any arbitrary matcher. But then I realized it will require an additional effort to verify how this veto vote (e.g. by incompatible country in this implementation) will affect the outcome of other matchers and how many valid matches could be removed. Usually we decide to create a dedicated ticket in such case so may I will just report it and hope for some dev resources in future to take care of it. It looks like it may improve the quality quite severely.

Reply:
Just did it: https://github.com/openaire/iis/issues/1341